pkg/registry/apis/identity/team/store_members.go

```diff
@@ -0,0 +1,111 @@
+package team
+
+import (
+	"context"
+	"net/http"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/registry/rest"
+
+	"github.com/grafana/authlib/claims"
+	"github.com/grafana/grafana/pkg/api/dtos"
+	identityv0 "github.com/grafana/grafana/pkg/apis/identity/v0alpha1"
+	"github.com/grafana/grafana/pkg/registry/apis/identity/common"
+	"github.com/grafana/grafana/pkg/registry/apis/identity/legacy"
+	"github.com/grafana/grafana/pkg/services/apiserver/endpoints/request"
+	"github.com/grafana/grafana/pkg/setting"
+)
+
+var (
+	_ rest.Storage         = (*LegacyTeamMemberStore)(nil)
+	_ rest.Scoper          = (*LegacyTeamMemberStore)(nil)
+	_ rest.StorageMetadata = (*LegacyTeamMemberStore)(nil)
+	_ rest.Connecter       = (*LegacyTeamMemberStore)(nil)
+)
+
+func NewLegacyTeamMemberStore(store legacy.LegacyIdentityStore) *LegacyTeamMemberStore {
```

Comment:
maybe a name without "Store" in it -- since this does not actually store anything?

Reply:
Yes that makes sense. I will look at what we have used for other sub resources